Thalia/gp.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<GroupParameters xmlns="http://www.zurich.ibm.com/security/idemix" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.zurich.ibm.com/security/idemix GroupParameters.xsd">
+  <References>
+    <SystemParameters>http://thalia.nu</SystemParameters>
```

Comment:
This should be a url to the gp.xml file, prefixed by the Thalia/description.xml baseURL.
It doesn't need to actually exist, but needs to be "valid" in that sense.